src/app/app.module.ts

```diff
@@ -26,7 +29,10 @@ import { DialogErrorComponent } from './shared/dialog-error.component';
     ProfileComponent,
     EditProfileComponent,
     ActiveProfileComponent,
-    DialogComponent
+    DialogComponent,
+    BottomSheetLegendComponent,
+    ProfileCardComponent,
+    AdminComponent
```

Comment:
Can they be alphabetically ordered?

Reply:
This is now done for all the modules will commit soon